ongame.py

```diff
@@ -0,0 +1,221 @@
+            except BaseException:
+                print("===================")
+                print("get_frame() error! Reboot OAK-D.")
+                print("If reboot occur frequently, Bandwidth may be too much.")
+                print("Please lower FPS.")
+                print("==================")
+                break
+                
+            if(data["button_c"]==True):
+                m5.set_display_text(text="ゲーム終了", text_color=Colors.GREEN)
+                play_audio_sync("finish")
+                joints.move_joint_positions(pan=0, tilt=0.1)
+                OnGame = False
+                oakd_tracking_yolo.close()
+                sys.exit(0)  # 親プロセスに「終了」を通知
+
+
+            if OnGame is True:
+                #首の動き
+                if current_time - last_move_time > move_interval:
+                    if toggle:
+                        play_audio_sync("1")
+                        joints.move_joint_positions(pan=0, tilt=0.1)
+                        detection_start_time = time.time() + detection_delay
+                        m5.set_display_text(text="検知中", text_color=Colors.GREEN)
+                        is_detecting = True
+                        move_interval = operating_time
+                    else:
+                        joints.move_joint_positions(pan=0, tilt=-0.3)
+                        detection_start_time = None
+                        m5.set_display_text(text="停止中", text_color=Colors.BLUE)
+                        is_detecting = False
+                        move_interval = waiting_time
+                        User_Moved = False
+                    toggle = not toggle
+                    last_move_time = current_time
+            #動きの検出
+            if tracklets is not None:
+                for tracklet in tracklets:
+                    if tracklet.status.name == "TRACKED" and tracklet.label == 0:#jsonのlabels内でparsonは０番目
+                        id = tracklet.id
+                        x = tracklet.spatialCoordinates.x
+                        y = tracklet.spatialCoordinates.y
+                        z = tracklet.spatialCoordinates.z
+                        
+                        if detection_start_time is not None and current_time >= detection_start_time and id in previous_positions:
+                            prev_x, prev_y, prev_z = previous_positions[id]
+                            dx = abs(x - prev_x)
+                            dy = abs(y - prev_y)
+                            dz = abs(z - prev_z)
+                            if dx > 50 or dy > 50 or dz > 50:
+                                if is_detecting and not User_Moved:
+                                    print(f"[WARNING] Person {id} moved! Δx={dx}, Δy={dy}, Δz={dz}")
+                                    play_audio_async("move")
+                                    User_Moved = True
+                                    m5.set_display_text(text="動いた！", text_color=Colors.RED)
```

Comment:
```suggestion
                                    m5.set_display_text(text="動いた!", text_color=Colors.RED)
```
全角の！は表示エラーになっているので半角に変えていただきたいです。